scripts/create_new_jsons.py

```diff
@@ -0,0 +1,208 @@
+"""create_new_jsons
```

Comment:
The code looks great, I do wonder whether the name could be improved? E.g `create_marketplace_json`?

Reply:
Done :)